src/backend/web/main.py

```diff
@@ -153,6 +156,7 @@
 app.register_blueprint(suggestion_blueprint)
 app.register_blueprint(suggestion_review_blueprint)
 app.register_blueprint(team_admin)
+app.register_blueprint(webhooks)
```

Comment:
I think there's tests for adding these blueprints? I forget

Reply:
I couldn't find any specific tests, but I think test_webhooks should verify that that the routes are registered correctly.